index.bs

```diff
@@ -1211,7 +1211,7 @@ partial interface MLGraphBuilder {
     *output size = (input size - 1) ** *stride + filter size + (filter size - 1) ** *(dilation - 1) - beginning padding - ending padding + output padding*
 </div>
 
-### element-wise binary operations ### {#api-mlgraphbuilder-binary}
+### Element-wise binary operations ### {#api-mlgraphbuilder-binary}
```

Comment:
Should it also mention "method"? such as "The methods for element-wise binary operations"?

Reply:
For consistency keep the top-level group as "Element-wise unary operations" and have "The x() method" subsections for each of these ops where you define "The x() method steps are:".

Reply:
Yes, for consistency we should do that unless too much boilerplate makes it actually harder to read.

Keep the current names in this PR and push separate PRs for "element-wise binary operations" and "element-wise unary operations".

Smaller self-contained PRs are easier to review. We can then see how it'll look like and which way to go.

Reply:
I think this would expand and clutter the spec way too much, with little added benefit.
What matters in all these methods is how the inputs and outputs are treated between the script and implementation and then the underlying platform.
These ops might be mapped directly to the underlying platform, meaning basically empty shell algorithms (and sections), not adding any useful information.
While we could do these changes, looks lower priority compared to other work on the spec, so I suggest we leave this to a later stage. When done, will be (easily) done as suggested.

Reply:
Sounds good.